app/models/spree/user_decorator.rb

```diff
@@ -4,7 +4,8 @@
   devise :omniauthable
 
   def apply_omniauth(omniauth)
-    if %w(facebook google_oauth2).include? omniauth['provider']
+    skip_signup_providers = SpreeSocial::OAUTH_PROVIDERS.map{ |p| p[1] if p[2] == "true" }.compact
+    if skip_signup_providers.include? omniauth['provider']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.